runtime/mainnet/src/lib.rs

```diff
@@ -1477,11 +1250,7 @@ construct_runtime!(
 		BaseFee: pallet_base_fee,
 		HotfixSufficients: pallet_hotfix_sufficients,
 
-		Claims: pallet_airdrop_claims,
-		Roles: pallet_roles,
-		Jobs: pallet_jobs,
-		Dkg: pallet_dkg,
-		ZkSaaS: pallet_zksaas,
```

Comment:
ideally we need a migration to remove these pallets data not sure if it will work without since the pallet indexes will change

Reply:
Tracked in #704

Reply:
Let's just zero out these indices and add services in higher indices if it doesn't work.